.github/workflows/build.yml

```diff
@@ -250,7 +250,7 @@ jobs:
         runner:
           - ubuntu-latest
           # - macos-latest
-        node: [18, 19]
+        node: [18]
```

Comment:
Let's reinstate our support of Node 19 - since sooner or later this is going to break if we stop testing it. Both @Chriscbr and @MarkMcCulloh offered to help you with that and I'm happy to assist too if this gets complicated

Reply:
Or actually if this gets complicated beyond Mark and Chris's skills I probably won't be able to help :)
But I'll find someone who can in that case...

Reply:
Going to put a PR up for it in just a moment. I think we can just update the cli.test.ts to ignore engines. testing now.

Reply:
@staycoolcall911 you will find the e2e re-enabled and above comments addressed in: https://github.com/winglang/wing/pull/1087